spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskDefinitionController.java

```diff
@@ -113,9 +131,50 @@ public TaskDefinitionResource save(@RequestParam("name") String name,
 	}
 
 	/**
+	 * Register a composed task definition for future execution.
+	 *
+	 * @param name of the composed task
+	 * @param graph DSL definition for the composed task
+	 */
+	@RequestMapping(value = "/compose", method = RequestMethod.POST)
```

Comment:
It is.
But the fact that it is composed could be captured in there.
Also, but that is a purely technical excuse, having a dedicated resource class (that may be a subclass of the former that actually adds nothing) helps with `RootController`'s `entityLinks.linkToCollectionResource()`